proj/src/main/webapp/script.js

```diff
@@ -428,6 +428,35 @@ function addUsernameToMessage(uid, elementId) {
     })
 }
 
+const bioBox = document.getElementById('user-bio');
+const editInputBox = document.getElementById('bio-edit');
+
+bioBox.addEventListener('dblclick', function() {
+    const bioBox = document.getElementById('user-bio');
```

Comment:
You can also do 
`bioBox.addEventListener('dblclick', function(event) {
    const bioBox = event.target;}` But it looks like `this` might also give you a reference to `bioBox` so you don't need to search the document for it.

Reply:
fixed